sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/DirectConnectionConfig.java

```diff
@@ -0,0 +1,269 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.cosmos;
+
+import java.time.Duration;
+import java.util.List;
+
+/**
+ * Represents the connection config with {@link ConnectionMode#DIRECT} associated with Cosmos Client in the Azure Cosmos DB database service.
+ */
+public final class DirectConnectionConfig extends ConnectionConfig {
+    //  Constants
+    private static final Duration DEFAULT_IDLE_CHANNEL_TIMEOUT = Duration.ZERO;
```

Comment:
Will ZERO default to FX one. If so we might not have control on it. Why not be explicit?

Reply:
https://github.com/Azure/azure-sdk-for-java/issues/10983